### DISCUSSION TITLE
Simple server backup workflow with age?

### DISCUSSION THREAD

**Discussion by Brixy** [FiloSottile/age#528] [General]
Hello guys,

I am not a crypto expert but have been using age for several years, particularly as a `pass` alternative and for file exchange. Thanks very much for this great tool.

For server backups of project folders I currently use a simple `gocryptfs` workflow:

```
mkdir mnt-encrypted mnt-remote
gocryptfs -reverse my-folder mnt-encrypted
sshfs user@server.url mnt-remote
rsync […]
```

If possible I would like to replace gocrypfs with age.

- Does anyone use an age workflow for a similar use case, and how exactly?
- Why do you prefer age to other tools for this use case?

Thank you very much!

**Comment by RollinRolanding** [comment]
I am also no crypto expert but I incorporated `age` into my backup process. Basically my script does the below followed by a bunch AWS S3 multipart upload fun:
```
tar -c --use-compress-program=pigz /something | age -r <me>
```
I'm using `pigz` in place of `tar cz` i.e., `gzip` because I'm archiving fairly large directories, ~2-3TB.

If you also need to chunk the encrypted files, you can pipe the above to e.g., ` tar ... | age ... | split -b $SPLIT_SIZE - "${FILE}.part-"`

**Reply by Brixy** [reply]
Hello @RollinRolanding,

excellent! Thanks a lot for your advice and the code example. That looks very promising. I will play with it and try to integrate your code into my workflow.

Restic and other excellent tools seems to do 'too much'. A simple and future proof age workflow for encrypted backups seems very sensible to me.

Thank you very much
